utils/compare_ds.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python3
+
+import argparse
+import sys
+import xml.etree.ElementTree as ET
+import difflib
+
+import ssg.constants
+
+ns = {
+    "ds": "http://scap.nist.gov/schema/scap/source/1.2",
+    "xccdf": "http://checklists.nist.gov/xccdf/1.2",
+    "oval": "http://oval.mitre.org/XMLSchema/oval-definitions-5",
+    "catalog": "urn:oasis:names:tc:entity:xmlns:xml:catalog",
+    "xlink": "http://www.w3.org/1999/xlink",
+}
+remediation_type_to_uri = {
+    "bash": "urn:xccdf:fix:script:sh",
+    "ansible": "urn:xccdf:fix:script:ansible",
+    "puppet": "urn:xccdf:fix:script:puppet",
+    "anaconda": "urn:redhat:anaconda:pre",
+}
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Compares two datastreams with regards to presence of"
+        "OVAL checks and all remediations")
+    parser.add_argument(
```

Comment:
Make the old and new DS filenames required positional arguments rather than required optional arguments. The current approach is somehow [non-standard](https://docs.python.org/3/library/argparse.html#required).